features/step_definitions/common_claim_steps.rb

```diff
@@ -138,18 +138,18 @@
 end
 
 When(/^I click "Continue" in the claim form and move to the '(.*?)' form page$/) do |page_title|
-  original_title = page.title
+  original_header = page.first('h2').text
```

Comment:
WHy move away from the page object model?

Reply:
I am actually reverting to what it was before my prior commit and page object model will be revisited in future.